internal/driver/controllerserver_helper.go

```diff
@@ -677,33 +650,6 @@ func (cs *ControllerServer) getInstance(ctx context.Context, linodeID int) (*lin
 	return instance, nil
 }
 
```

Comment:
Is there really no value in having these checks client side at all?

Reply:
I tend to prefer to rely on API internal logic and validation and for CSI to act as a simple pass-through.

The only problem could be a rate-limiting issue on API (GET vs. POST).
But attacher already has a backoff on error
https://github.com/kubernetes-csi/external-attacher?tab=readme-ov-file#csi-error-and-timeout-handling

But that may be questionable, or we may have to wait for linodego to implement rate-limiting.
That's why this PR was in draft state at first.